src/client/index.js

```diff
@@ -257,6 +266,87 @@ export class BncClient {
     return this._broadcastDelegate(signedTx)
   }
 
+  /**
+   * Create and sign a multi send tx
+   * @param {String} fromAddress
+   * @param {Array} transfers
+   * @example
+   * const transfers = [
+   * {
+   *   "to": "tbnb1p4kpnj5qz5spsaf0d2555h6ctngse0me5q57qe",
+   *   "coins": [{
+   *     "denom": "BNB",
+   *     "amount": 10
+   *   }]
+   * },
+   * {
+   *   "to": "tbnb1scjj8chhhp7lngdeflltzex22yaf9ep59ls4gk",
+   *   "coins": [{
+   *     "denom": "BTC",
+   *     "amount": 10
+   *   }]
+   * }]
+   * @param {String} memo optional memo
+   * @param {Number} sequence optional sequence
+   * @return {Promise} resolves with response (success or fail)
+   */
+  async multiSend(fromAddress, transfers, memo="", sequence=null) {
+    if(!fromAddress) {
+      throw new Error("fromAddress should not be falsy")
+    }
+
+    if(!Array.isArray(transfers)) {
+      throw new Error("transfers should be array")
+    }
+
+    checkTransfers(transfers)
+
+    transfers.forEach(item=>{
+      item.coins.forEach(coin=>{
+        coin.amount = Number(coin.amount * Math.pow(10, 8))
+      })
+    })
+
+    const fromAddrCode = crypto.decodeAddress(fromAddress)
+    const inputs = [{ address: fromAddrCode, coins: [] }]
+    const outputs = []
+
+    transfers.forEach((item)=>{
+      const toAddcCode = crypto.decodeAddress(item.to)
+      inputs[0].coins = inputs[0].coins.concat(item.coins)
+      outputs.push({address: toAddcCode, coins: item.coins})
```

Comment:
This logic should be a little bit more complicated to deal with multiple outputs for the same denom (token). The quantities of a single denom (token) should be added together in its corresponding input `coin` object.

In this example, the quantities of `PPC-00A` in the `outputs` have been summed in the corresponding input `coin` object:

https://testnet-dex.binance.org/api/v1/tx/5F37451A58EA3FA322A102FE1B9EAF9A0FFFB5B92CB29285E13227557E652930?format=json

As you can see, there is one `coin` in the single `input` with a quantity of `55859245907931`, but this code would produce multiple `coin` objects each with a quantity of `55915161069`.